lib/Installation/ConfirmInstallation.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Confirm Installation Page.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Installation::ConfirmInstallation;
+use Installation::Popups::AcceptPopup;
+use Installation::Popups::OKPopup;
+use strict;
+use warnings;
+use testapi qw(save_screenshot record_info);
```

Comment:
We don't include testapi in the framework (unless there is not other way to interact with the screen, very specific corner cases), please remove it, you have to do this operations from test layer.